production/helm/loki/values.yaml

```diff
@@ -332,6 +333,7 @@ loki:
       service: "memcached-client"
       batch_size: 256
       parallelism: 10
+      timeout: "500ms"      
```

Comment:
The default in Loki is 100ms. Can you set that here? It can then be overwritten when you deploy it in your cluster.

Reply:
I changed it as you said, please confirm. thank you!